python/doc/examples/reliability_sensitivity/reliability/plot_nais.py

```diff
@@ -0,0 +1,232 @@
+"""
+Non parametric Adaptive Importance Sampling (NAIS)
+==================================================
+"""
+# %%
+#
+# The objective is to evaluate a probability from the Non parametric Adaptive Importance Sampling (NAIS) technique.
+#
+# We consider the four-branch function :math:`g : \mathbb{R}^2 \rightarrow \mathbb{R}` defined by:
+#
+# .. math::
+#   \begin{align*}
+#   g(\vect{X}) = \min \begin{pmatrix}5+0.1(x_1-x_2)^2-\frac{(x_1+x_2)}{\sqrt{2}}\\
+#   5+0.1(x_1-x_2)^2+\frac{(x_1+x_2)}{\sqrt{2}}\\
+#   (x_1-x_2)+ \frac{9}{\sqrt{2}}\\
+#   (x_2-x_1)+ \frac{9}{\sqrt{2}}
+#   \end{pmatrix}
+#   \end{align*}
+#
+# and the input random vector :math:`\vect{X} = (X_1, X_2)` which follows the standard 2-dimensional Normal distribution:
+#
+# .. math::
+#   \begin{align*}
+#   \vect{X} \sim  \mathcal{N}(\mu = [0, 0], \sigma = [1,1], corr = \mat{I}_2)
+#   \end{align*}
+#
+# We want to evaluate the probability:
+#
+# .. math::
+#   \begin{align*}
+#   p = \mathbb{P} ( g(\vect{X}) \leq 0 )
+#   \end{align*}
+#
+
+# %%
+# First, import the python modules:
+
+# %%
+import openturns as ot
+from openturns.viewer import View
+import math
+
+# %%
+# Create the probabilistic model :math:`Y = g(\vect{X})`
+# -----------------------------------------------
```

Comment:
```suggestion
# Create the probabilistic model :math:`Y = g(\vect{X})`
# ------------------------------------------------------
```
Because CI has warnings treated as errors, this prevents the doc compilation.